### PR TITLE
tests added

### PR DESCRIPTION
**Pull request by DimaCoDeeema**

### CODE REVIEW

File: src/getCoinCombination.test.js [hunk 1]
```diff
@@ -7,4 +7,32 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return an array`, () => {
+    expect(getCoinCombination(1))
+      .toEqual([
+        1, 0, 0, 0,
+      ]);
```
Comment (redsky2410) [inline src/getCoinCombination.test.js:15]: it is a checks from example. Add your own

**Review by redsky2410 — CHANGES_REQUESTED** [review]
Pay attention on the comments

File: src/getCoinCombination.test.js (hunk 1 above)
Comment (DimaCoDeeema) [reply src/getCoinCombination.test.js:15]: done

File: src/getCoinCombination.test.js [hunk 2]
```diff
@@ -7,4 +7,32 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return an array`, () => {
```
[the remark this answers is not in the document]
Comment (DimaCoDeeema) [reply src/getCoinCombination.test.js:11]: done

**Pull request closed without merging**